lib/app/modules/Home/views/local_widgets/home_character_hp_xp_view.dart

```diff
@@ -1,40 +1,108 @@
-  const HomeCharacterHpExpView({Key? key}) : super(key: key);
+  const HomeCharacterHpExpView({super.key});
+
+  @override
+  Widget build(BuildContext context) => Row(
+    children: [
+      Expanded(
+        child: InkWell(
+          splashColor: Theme.of(context).splashColor,
+          borderRadius: BorderRadius.circular(10),
+          onTap: () => Get.dialog(const HPDialog()),
+          child: const Padding(
+            padding: EdgeInsets.all(4),
+            child: HpBar(),
+          ),
+        ),
+      ),
+      const SizedBox(width: 16),
+      const Expanded(
+        child: _EXP(),
+      ),
+    ],
+  );
+}
+
+class _EXP extends StatefulWidget {
+  const _EXP();
 
+  @override
+  State<_EXP> createState() => _EXPState();
+}
+
+class _EXPState extends State<_EXP> with CharacterServiceMixin {
+  var _hidden = true;
+
+  Timer _timer = Timer(Duration.zero, () {});
+  
+  void showButtons({bool shown = true}) {
+    setState(() {_hidden = !shown;});
+    _timer.cancel();
+    if (!shown) return;
+    _timer = Timer(const Duration(seconds: 2), () {
+      setState(() {_hidden = true;});
+    });
+  }
+  
   @override
   Widget build(BuildContext context) {
-    return Row(
+    return Stack(
       children: [
-        Expanded(
-          child: InkWell(
-            splashColor: Theme.of(context).splashColor,
-            child: const Padding(
-              padding: EdgeInsets.all(4),
-              child: HpBar(),
-            ),
-            borderRadius: BorderRadius.circular(10),
-            onTap: () => Get.dialog(const HPDialog()),
+        InkWell(
+          splashColor: Theme.of(context).splashColor,
+          borderRadius: BorderRadius.circular(10),
+          onTap: showButtons,
+          child: const Padding(
+            padding: EdgeInsets.all(4),
+            child: ExpBar()
           ),
         ),
-        const SizedBox(width: 16),
-        Expanded(
-          child: InkWell(
-            splashColor: Theme.of(context).splashColor,
-            child: const Padding(
-              padding: EdgeInsets.all(4),
-              child: ExpBar(),
+        IgnorePointer(
+          ignoring: _hidden,
+          child: AnimatedOpacity(
+            opacity: _hidden ? 0 : 1,
+            duration: const Duration(milliseconds: 200),
+            child: Row(
+              children: [
+                const SizedBox(width: 16),
+                PrimaryChip(
+                  label: '+1',
+                  onPressed: () {
+                    showButtons(shown: false);
+                    charService.updateCharacter(
+                      char.copyWith(
+                        stats: char.stats.copyWith(
+                          currentXp: 1 + (char.stats.currentXp),
+                        ),
+                      ),
+                    );
+                  },
+                ),
+                const Expanded(child: SizedBox(width: 0)),
+                PrimaryChip(
+                  label: '',
```

Comment:
Does this not still create some padding between the icon and label? We might want to add some label here

Reply:
No, try it out. It looks nice.

Reply:
Wait. It might do that slightly, the thing seems off center. But there should not be a label. I will investigate.

Reply:
Done